sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2943,3 +2943,73 @@ case class Sentences(
     copy(str = newFirst, language = newSecond, country = newThird)
 
 }
+
+/**
+ * Splits a given string by a specified delimiter and return splits into a
```

Comment:
Can we highlight the difference of this expression from `StringSplit`?

Reply:
Done. Hope my words clarify it.